site/cypress/e2e/Cypress-System/autograding_status_2.spec.js

```diff
@@ -7,7 +7,7 @@ skipOn(Cypress.env('run_area') === 'CI', () => {
     describe('Pre autograding test', () => {
         before(() => {
             cy.login();
-            cy.wait(500);
+            // cy.wait(500);
```

Comment:
Should this be removed instead of commented out?

Reply:
Fixed in https://github.com/Submitty/Submitty/pull/10562/commits/8107197e747bb610b3ae3ba2ba25d659559ebf33